extensions/wikia/ARecoveryEngine/templates/ARecoveryEngineApiController_getBootstrap.mustache

```diff
@@ -4,11 +4,16 @@ window._sp_ = {
     config: {
         enable_rid: true,
         content_control_callback: function() {
-            var msg = '<h2>Ad blocker interference detected!</h2>';
+            var logGroup = 'ext.wikia.aRecoveryEngine.recovery.bootstrap',
+                msg = '<h2>Ad blocker interference detected!</h2>';
                 msg += '<h3>If you have added a new rule to your ad blocker that interferes with the loading of ads on your pages, ';
                 msg += 'this Interference Screen is the expected behaviour. <br />';
                 msg += 'Removing the rule(s) should load the page as expected.</h3>';
```

Comment:
I agree that there should be one message, but. There is an issue with that. Basically, we can not generate HTML tags trough CSS. IMO we should wait till we get a real message from ComDev and then think how to implement it in both places. I wouldn't touch it now to not duplicate work.